qtpy/QtLocation.py

```diff
@@ -8,18 +8,27 @@
 """Provides QtLocation classes and functions."""
 
 from . import (
+    parse,
     PYQT5,
     PYQT6,
     PYSIDE2,
     PYSIDE6,
+    PYQT_VERSION,
+    PYSIDE_VERSION,
```

Comment:
Instead of importing both of these separately, and particularly if this is related to the Qt version rather than the binding version, you could just import `QT_VERSION` and check that instead.

Reply:
I actually tried that first, and my editor insisted that `QT_VERSION` didn't exist.

This is also one of the modules that I need the environment matrix for. I think that PyQt6 had this module earlier than PySide6 did.